### PR TITLE
Remove initialiser and take the session as an argument

### PR DESCRIPTION
**Pull request by dylanmacdonald**

### CODE REVIEW

File: email/emailer.go [hunk 1]
```diff
@@ -12,16 +10,13 @@ import (
...
-// Init creates a new session for SES requests
-func Init() error {
-	var err error
-	awsSession, err = session.NewSession(&aws.Config{
-		Region:      aws.String("us-west-2"),
-		Credentials: credentials.NewEnvCredentials(),
-	})
-	return err
+// NewSession initialises an AWS session
+func NewSession(sesRegion string) (*session.Session, error) {
```
Comment (dylannz) [inline email/emailer.go]: Is this wrapper method really necessary?

Comment (dylanmacdonald) [reply email/emailer.go]: No, but it provides a default method of starting a session if the client wants to plug & play the package

Comment (dylannz) [reply email/emailer.go]: All that method is really doing now is hiding the call to `credentials.NewEnvCredentials()`. If you want to abstract away the AWS stuff, you should wrap the AWS session struct entirely.

Comment (dylannz) [reply email/emailer.go]: Something like this (`mailer` contains the AWS session)

```
mailer, err := email.NewClient(cfg...)
iferr...

mailer.Send(email.Email{
  To:      "",
  Subject: "",
  ...
})
```

Comment (proglottis) [reply email/emailer.go]: If you did this you could also make a "MustSend" version or equivalent that does logging and no error return too

File: email/emailer.go [hunk 2]
```diff
@@ -33,9 +32,37 @@ type Email struct {
...
+			},
+		},
+	}
+	_, err := svc.SendEmail(input)
+	return err
+}
+
+// MustSend sends a simple email via AWS SES
+// MustSend will log any errors that occur, but they will not be returned
+func (m *Mailer) MustSend(e Email) {
```
Comment (proglottis) [inline email/emailer.go]: This method could just use `Send` above and avoid the duplication

**Comment by dylannz** [comment]
Nice.

**Pull request merged**